stabilizer/src/main/java/com/hazelcast/stabilizer/worker/TestContextImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.hazelcast.stabilizer.worker;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.stabilizer.test.TestContext;
+
+class TestContextImpl implements TestContext {
+    private final String testId;
+    private final HazelcastInstance hazelcastInstance;
+
+    volatile boolean stopped = false;
```

Comment:
Why is this field not private?

Also the field by default is false, so you can remove = false.